org.civicrm.reqangular/test/mocks/services/api/contact-mock.js

```diff
@@ -1,232 +1,93 @@
+              } else if (filters[key].IN) {
+                return _.includes(filters[key].IN, contact[key]);
+              } else {
+                return contact[key] === filters[key];
+              }
+            });
+          });
+        }
 
-                if (filters) {
-                    list = list.filter(function (contact) {
-                        return Object.keys(filters).every(function (key) {
-                            if (key === 'display_name') {
-                                return (new RegExp(filters[key], 'i')).test(contact[key]);
-                            } else if (filters[key].IN) {
-                                return _.includes(filters[key].IN, contact[key]);
-                            } else {
-                                return contact[key] === filters[key];
-                            }
-                        });
-                    });
-                }
+        if (pagination) {
+          start = (pagination.page - 1) * pagination.size;
+          end = start + pagination.size;
 
-                if (pagination) {
-                    start = (pagination.page - 1) * pagination.size;
-                    end = start + pagination.size;
+          list = list.slice(start, end);
+        }
 
-                    list = list.slice(start, end);
-                }
+        return promiseResolvedWith({
+          list: list,
+          total: list.length,
+          allIds: list.map(function (contact) {
+            return contact.id;
+          }).join(',')
+        });
+      },
+      find: function (id, value) {
+        var contact = value || ContactData.all.values.filter(function (contact) {
+          return contact.id === id;
+        })[0];
 
-                return promiseResolvedWith({
-                    list: list,
-                    total: list.length,
-                    allIds: list.map(function (contact) {
-                        return contact.id;
-                    }).join(',')
-                })
-            },
-            find: function (id, value) {
-                var contact = value || this.mockedContacts().list.filter(function (contact) {
-                    return contact.id === id;
-                })[0];
+        return promiseResolvedWith(contact);
+      },
 
-                return promiseResolvedWith(contact);
-            },
+      /**
+       * Adds a spy on every method for testing purposes
+       */
+      spyOnMethods: function () {
+        _.functions(this).forEach(function (method) {
+          spyOn(this, method).and.callThrough();
+        }.bind(this));
+      },
 
-            /**
-             * Adds a spy on every method for testing purposes
-             */
-            spyOnMethods: function () {
-                _.functions(this).forEach(function (method) {
-                    spyOn(this, method).and.callThrough();
-                }.bind(this));
-            },
+      /**
+       * # DRAFT #
```

Comment:
Actually this method needs to go, i added a `@todo` to the method description. I just didn't do it in this PR cause it would add a bit more complications that I didn't need it for this ticket